hudi-utilities/src/main/java/org/apache/hudi/utilities/streamer/StreamSync.java

```diff
@@ -1037,6 +1037,11 @@ public void runMetaSync() {
 
       Map<String, HoodieException> failedMetaSyncs = new HashMap<>();
       for (String impl : syncClientToolClasses) {
+        if (impl.trim().isEmpty()) {
```

Comment:
would be better if we just filter out empty string when building `syncClientToolClasses`.